### PR TITLE
[WIP] Dutch translation

### PR DESCRIPTION
**Pull request by JesseZweers**

### CODE REVIEW

**Comment by Matthe072** [comment]
i think the main is good

**Comment by Matthe072** [comment]
made by JesseZweers and Matthe072

**Comment by Matthe072** [comment]
i am going to make it now

**Comment by Matthe072** [comment]
when you come back within 1 hour you kan merge it

File: nl-NL.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+"Refresh Icon Cache" = "Icon Cache refreshen"
+"Description:\n\n" = "Beschrijving:"
+"This option makes the jailbreak regenerate SpringBoard's system application installation cache to cause newly installed .app bundles to appear on the icon list." = "Deze optie zorgt ervoor dat de jailbreak SpringBoard's systeemapplicatie installatiecache hergenereert, zodat nieuw geinstalleerde .app bundles in de pictogrammenlijst verschijnen."
+"iOS 11.0-12.2 and 12.4 on arm64/arm64e SoCs A7-A12X." = "iOS 11.0-12.2 en 12.4 op arm64/arm64e SoCs A7-A12X."
+"Disable Updates" = "Updates uitschakelen"
+"This option makes the jailbreak effectively disable the system's software update mechanism to prevent the system from automatically upgrading to the latest available firmware which may not be supported by the jailbreak at that time." = "Met deze optie schakelt de jailbreak het updatemechanisme van het besturingssysteem uit, om te voorkomen dat het besturingssysteem automatisch een update uitvoert naar de nieuwste beschikbare firmware. Mogelijk wordt die firmware op dat moment niet door de jailbreak ondersteund."
```
Comment (SliceOLife) [inline nl-NL.lproj/Localizable.strings]: "Nieuwst" instead of nieuwste.

File: nl-NL.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+"iOS 11.0-12.1.2 on arm64/arm64e SoCs A7-A12X." = "iOS 11.0-12.1.2 op arm64/arm64e SoCs A7-A12X."
+"Disable Revokes" = "Revokes uitschakelen"
+"This option makes the jailbreak effectively disable the system's online certificate status protocol system to prevent enterprise certificates which the jailbreak may be signed with from getting revoked." = "Met deze optie schakelt de jailbreak het online certificaatstatusprotocol van het besturingssysteem uit, om te voorkomen dat enterprisecertificaten, waarmee de jailbreak ondertekent kan worden, worden ingetrokken."
+"iOS 11.0-11.4.1 on arm64 SoCs A7-A11." = "iOS 11.0-11.4.1 op arm64 SoCs A7-A11."
+"Set Boot Nonce" = "Boot Nonce instellen"
+"Description:" = "Beschrijving:"
+"This option makes the jailbreak set the persistent com.apple.System.boot-nonce variable in non-volatile random-access memory NVRAM which may be required to downgrade to an unsigned iOS firmware by using SHSH files." = "Met deze optie stelt de jailbreak de persistente variabele, com.apple.System.boot-nonce, in het non-volatile random-access geheugen NVRAM, dat mogelijk nodig is om te downgraden naar een niet-ondertekende iOS firmware met behulp van SHSH-bestanden."
```
Comment (SliceOLife) [inline nl-NL.lproj/Localizable.strings]: Geen komma nodig voor com.apple.System.boot-nonce.

niet-ondertekende -> niet gesigneerde

File: nl-NL.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+"iOS 11.0-12.1.2 on arm64/arm64e SoCs A7-A12X." = "iOS 11.0-12.1.2 op arm64/arm64e SoCs A7-A12X."
+"Export TFP0" = "TFP0 exporteren"
+"This option makes the jailbreak modify the host-port to grant any process access to the host-priv-port." = "Met deze optie wijzigt de jailbreak de host-poort om elk proces toegang te geven tot de host-priv-poort."
+"This option effectively grants any process access to the kernel task port TFP0 and allows re-jailbreaking without exploiting again." = "Deze optie verleent elk proces toegang tot de kerneltaakpoort TFP0 en staat het opnieuw jailbreaken toe zonder opnieuw te exploiteren."
+"This option is considered unsafe as the privilege this option effectively grants to processes can be used for bad purposes by malicous apps." = "Deze optie wordt als onveilig beschouwd, omdat het voorrecht dat deze optie aan processen verleent, door kwaadaardige applicaties gebruikt kan worden voor slechte doeleinden."
```
Comment (SliceOLife) [inline nl-NL.lproj/Localizable.strings]: voorrecht -> privilege

File: nl-NL.lproj/Localizable.strings [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+"iOS 11.0-12.1.2 on arm64/arm64e SoCs A7-A12X." = "iOS 11.0-12.1.2 op arm64/arm64e SoCs A7-A12X.",
+"Restore RootFS" = "RootFS herstellen"
+"This option makes the jailbreak restore the root filesystem RootFS to the snapshot which is created by the system when the device is restored." = "Met deze optie herstelt de jailbreak het rootbestandssysteem RootFS naar de snapshot die door het systeem wordt gemaakt wanneer het apparaat wordt hersteld."
+"This option effectively allows uninstalling the jailbreak without losing any user data." = "Met deze optie kun je de jailbreak verwijderen zonder gebruikersgegevens te verliezen."
+"iOS 11.0-12.1.2 on arm64/arm64e SoCs A7-A12X.", = "iOS 11.0-12.1.2 op arm64/arm64e SoCs A7-A12X.",
+"Max Memory Limit" = "Max geheugenlimiet"
```
Comment (SliceOLife) [inline nl-NL.lproj/Localizable.strings]: Max -> Maximale

File: nl-NL.lproj/Localizable.strings [hunk 5]
```diff
@@ -0,0 +1,369 @@
...
+"Memory allocation failure: %s" = "Geheugentoewijzingsfout: %s"
+"Unable to clean success file." = "Kan het succesbestand niet opschonen."
+"Unable to synchronize app preferences. Please restart the app and try again." = "Kan de voorkeuren niet synchroniseren. Start de applicatie opnieuw op en probeer het opnieuw."
+"Unable to create test file." = "Kan geen testbestand maken."
+"Unable to clean test file." = "Kan het testbestand niet opschonen."
+"Unable to inject trust cache" = "Kan de trust cache niet injecteren."
+"Exploiting kernel..." = "Kernel exploiteren..."
+"Unable to get host port." = "Kan hostpoort niet verkrijgen."
+"No exploit selected." = "Geen exploit geselecteerd."
+"Unable to exploit kernel. This is not an error. Reboot and try again." = "Kan kernel niet exploiteren. Dit is geen fout. Start de applicatie opnieuw op en probeer het opnieuw."
```
Comment (SliceOLife) [inline nl-NL.lproj/Localizable.strings]: "Start de applicatie opnieuw" could be shortened to "Herstart de applicatie"

File: nl-NL.lproj/Localizable.strings [hunk 6]
```diff
@@ -0,0 +1,369 @@
...
+"Unable to exit." = "Kan niet afsluiten."
+"Exploited kernel.\n" = "Kernel geëxploiteerd.\n"
+"Initializing jailbreak..." = "Jailbreak initialiseren..."
+"Unable to swap my process sandbox in kernel memory." = "Kan mijn processandbox niet in het kernelgeheugen omwisselen."
+"Unable to find kernel offsets." = "Kan kernel-offsets niet vinden."
+"Unable to find my process in kernel memory." = "Kan mijn proces niet vinden in het kernelgeheugen."
+"Unable to find kernel's credentials in kernel memory." = "Kan de credenties van de kernel niet vinden in het kernelgeheugen."
+"Unable to verify shenanigans in kernel memory." = "Kan shenanigans in het kernelgeheugen niet verifiëren."
+"Unable to overwrite shenanigans in kernel memory." = "Kan shenanigans in kernelgeheugen niet overschrijven."
+"Unable to steal kernel's credentials." = "Kan de credenties van de kernel niet stelen."
```
Comment (SliceOLife) [inline nl-NL.lproj/Localizable.strings]: Should be "Niet in staat om de credenties van de kernel te stelen"

File: nl-NL.lproj/Localizable.strings [hunk 7]
```diff
@@ -0,0 +1,369 @@
...
+"Unable to exploit kernel. This is not an error. Reboot and try again." = "Kan kernel niet exploiteren. Dit is geen fout. Start de applicatie opnieuw op en probeer het opnieuw."
+"Unable to exit." = "Kan niet afsluiten."
+"Exploited kernel.\n" = "Kernel geëxploiteerd.\n"
+"Initializing jailbreak..." = "Jailbreak initialiseren..."
+"Unable to swap my process sandbox in kernel memory." = "Kan mijn processandbox niet in het kernelgeheugen omwisselen."
+"Unable to find kernel offsets." = "Kan kernel-offsets niet vinden."
+"Unable to find my process in kernel memory." = "Kan mijn proces niet vinden in het kernelgeheugen."
+"Unable to find kernel's credentials in kernel memory." = "Kan de credenties van de kernel niet vinden in het kernelgeheugen."
+"Unable to verify shenanigans in kernel memory." = "Kan shenanigans in het kernelgeheugen niet verifiëren."
+"Unable to overwrite shenanigans in kernel memory." = "Kan shenanigans in kernelgeheugen niet overschrijven."
```
Comment (SliceOLife) [inline nl-NL.lproj/Localizable.strings]: Should be "Niet in staat om shenanigans in het kernelgeheugen te overschrijven"

It's up for debate if you want to directly translate shenanigans to it's Dutch counterpart "geintjes". I'll leave that up to you.

**Comment by JesseZweers** [comment]
@SliceOLife I've implemented your suggested changes. Thank you!

@Samgisaninja Everything has been translated. I'll look over it tomorrow to make sure it's okay. And maybe @SliceOLife could also check it again if he has the time to do so. I'll reply here when everything is OK and ready to be merged.

**Comment by Matthe072** [comment]
We have done this

Op di 27 aug. 2019 om 23:51 schreef Jesse <notifications@github.com>

**Comment by SliceOLife** [comment]
@JesseZweers Looks good so far! I'll check out the rest tonight.

**Comment by Matthe072** [comment]
And is it ok?

**Comment by Matthe072** [comment]
And when come the A12 jailbreak

**Comment by Samgisaninja** [comment]
can y'all not delete the example files please and reply once those are restored?

**Comment by Samgisaninja** [comment]
also in the future, I will be blocking anyone who asks for an ETA. I don't know, and even if I did, I'm not allowed to comment on that matter, and even if I were, I wouldn't.

File: nl-NL.lproj/Localizable.strings (hunk 7 above)
Comment (Chibibowa) [reply nl-NL.lproj/Localizable.strings]: lol xD

**Comment by JesseZweers** [comment]
@Samgisaninja I've restored the example files. Everything looks good and you should be able to merge now.

**Comment by Samgisaninja** [comment]
kk thanks

**Pull request merged**